pkg/kubelet/apis/config/v1beta1/defaults_test.go

```diff
@@ -0,0 +1,216 @@
+				VolumeStatsAggPeriod:                      v1.Duration{Duration: time.Minute},
+				CgroupsPerQOS:                             utilpointer.BoolPtr(true),
+				CgroupDriver:                              "cgroupfs",
+				CPUManagerPolicy:                          "none",
+				CPUManagerReconcilePeriod:                 v1.Duration{Duration: 10 * time.Second},
+				MemoryManagerPolicy:                       v1beta1.NoneMemoryManagerPolicy,
+				TopologyManagerPolicy:                     v1beta1.NoneTopologyManagerPolicy,
+				TopologyManagerScope:                      v1beta1.ContainerTopologyManagerScope,
+				RuntimeRequestTimeout:                     v1.Duration{Duration: 2 * time.Minute},
+				HairpinMode:                               v1beta1.PromiscuousBridge,
+				MaxPods:                                   110,
+				PodPidsLimit:                              utilpointer.Int64(-1),
+				ResolverConfig:                            utilpointer.String(kubetypes.ResolvConfDefault),
+				CPUCFSQuota:                               utilpointer.BoolPtr(true),
+				CPUCFSQuotaPeriod:                         &v1.Duration{Duration: 100 * time.Millisecond},
+				NodeStatusMaxImages:                       utilpointer.Int32Ptr(50),
+				MaxOpenFiles:                              1000000,
+				ContentType:                               "application/vnd.kubernetes.protobuf",
+				KubeAPIQPS:                                utilpointer.Int32Ptr(5),
+				KubeAPIBurst:                              10,
+				SerializeImagePulls:                       utilpointer.BoolPtr(true),
+				EvictionHard:                              DefaultEvictionHard,
+				EvictionPressureTransitionPeriod:          v1.Duration{Duration: 5 * time.Minute},
+				EnableControllerAttachDetach:              utilpointer.BoolPtr(true),
+				MakeIPTablesUtilChains:                    utilpointer.BoolPtr(true),
+				IPTablesMasqueradeBit:                     utilpointer.Int32Ptr(DefaultIPTablesMasqueradeBit),
+				IPTablesDropBit:                           utilpointer.Int32Ptr(DefaultIPTablesDropBit),
+				FailSwapOn:                                utilpointer.BoolPtr(true),
+				ContainerLogMaxSize:                       "10Mi",
+				ContainerLogMaxFiles:                      utilpointer.Int32Ptr(5),
+				ConfigMapAndSecretChangeDetectionStrategy: v1beta1.WatchChangeDetectionStrategy,
+				EnforceNodeAllocatable:                    DefaultNodeAllocatableEnforcement,
+				VolumePluginDir:                           DefaultVolumePluginDir,
+				Logging: componentbaseconfigv1alpha1.LoggingConfiguration{
+					Format: "text",
+				},
+				EnableSystemLogHandler:  utilpointer.BoolPtr(true),
+				EnableProfilingHandler:  utilpointer.BoolPtr(true),
+				EnableDebugFlagsHandler: utilpointer.BoolPtr(true),
+				SeccompDefault:          utilpointer.BoolPtr(false),
+				MemoryThrottlingFactor:  utilpointer.Float64Ptr(DefaultMemoryThrottlingFactor),
+			},
+		},
+	}
```

Comment:
To cover more case

- empty is neccessage
- all zero(0,"",0s,false) is a negative case
- all settings(1,"xxx",60s,true) is positive case.

Reply:
@pacoxu I added some more tests so would you review the PR again? 🙏